src/Network/Response.php

```diff
@@ -559,9 +563,6 @@ protected function _setCookies()
      */
     protected function _setContentType()
     {
-        if (in_array($this->_status, [304, 204])) {
-            return;
-        }
```

Comment:
Right now a content-type will be emitted. I'm planning on changing the status code methods so that they clear the content-type though. I'm not entirely sure that linking the status code and content-type header is transparent to the end user though.